features/202-session-management/api.md

```diff
@@ -21,32 +21,12 @@ interface Session {
     createdByIP: string;
     lastAccessedIP: string;
     userAgent: UserAgent;
-    name: string;
-    data: object;
 }
 
 interface ExtraSessionInfoOptions {
-    collectDeviceName: boolean; // indicate SDK should collect device name information
-    deviceName?: string; // if falsy, SDK would try to detect it if possible
+    deviceName?: string;
```

Comment:
@kiootic can you add in the spec, client side should provide helper function to auto-gen the deviceName?